x-pack/index.js

```diff
@@ -37,6 +37,7 @@ import { translations } from './plugins/translations';
 import { upgradeAssistant } from './plugins/upgrade_assistant';
 import { uptime } from './plugins/uptime';
 import { ossTelemetry } from './plugins/oss_telemetry';
+import { encryptedSavedObjects } from './plugins/encrypted_saved_objects';
```

Comment:
__note:__ as always, naming is hard. This one may be a bit misleading, or may be not. Happy to accept any other naming suggestions.